crates/ruff_python_formatter/tests/snapshots/format@statement__function.py.snap

```diff
@@ -1014,4 +1020,12 @@ def function_with_one_argument_and_a_keyword_separator(
     *, argument: str
 ) -> ReallyReallyReallyReallyReallyReallyReallyReallyLongName:
     pass
+
+
+# PEP 646 introduced type var tuple in parameter annotation
+# https://peps.python.org/pep-0646/#change-2-args-as-a-typevartuple
+def function_with_variadic_generics(*args: *tuple[int]): ...
+def function_with_variadic_generics(
+    *args: *tuple[int],
```

Comment:
Is it intentional that you test here what happens if you have a trailing parameter comma? Should it instead test if there's a trailing tuple comma (in the subscript?)